contracts/Interface/IImplementationAuthority.sol

```diff
@@ -0,0 +1,7 @@
+// SPDX-License-Identifier: GPL-3.0
+
+pragma solidity ^0.6.9;
+
+interface IImplementationAuthority {
```

Comment:
This file is useless, and incorrect. ImplementationAuthority also has other methods such as `setImplementation`.
As this file is imported only in proxies, but directly this declaration in the proxy contract file.